Libraries/ActionSheetIOS/ActionSheetIOS.js

```diff
@@ -25,6 +26,18 @@ var ActionSheetIOS = {
       typeof callback === 'function',
       'Must provide a valid callback'
     );
+
+    if (options.tintColor) {
+      invariant(
+        typeof options.tintColor === 'string' && options.tintColor !== '',
+        'Must provide a valid string for tintColor'
+      );
+    }
+
+    if (processColor(options.tintColor)) {
+      Object.assign(options, { tintColor: processColor(options.tintColor) });
```

Comment:
Instead of Object.assign here, just do `options.tintColor = processColor(options.tintColor)`. Simpler code and lets us avoid an object allocation (not super important, but why not?)